### PR TITLE
feat: replace goose by handcradfted library

### PR DESCRIPTION
**Pull request by gfyrag**

### CODE REVIEW

File: components/orchestration/cmd/serve.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func newServeCommand() *cobra.Command {
 				fx.Invoke(func(lifecycle fx.Lifecycle, db *bun.DB) {
 					lifecycle.Append(fx.Hook{
 						OnStart: func(ctx context.Context) error {
-							return storage.Migrate(db, viper.GetBool(service.DebugFlag))
+							return storage.Migrate(db)
```
Comment (paul-nicolas) [inline components/orchestration/cmd/serve.go:45]: Why removing the debug flag ?

Comment (paul-nicolas) [reply components/orchestration/cmd/serve.go:45]: I said nothing, it was for goose, which is removed now

Comment (gfyrag) [reply components/orchestration/cmd/serve.go:45]: Goose support this, i have not reimplemented it.
Do you think I should?

Comment (paul-nicolas) [reply components/orchestration/cmd/serve.go:45]: It depends if we rely on it to debug migrations at first, but in any case I would say yes we should reimplement it (maybe later if needed if you're block for other things)

**Review by paul-nicolas — APPROVED** [review]

**Review by antoinegelloz — APPROVED** [review]

**Review by paul-nicolas — APPROVED** [review]

**Pull request merged**